ui/imports/utils/Constants.qml

```diff
@@ -904,6 +904,10 @@ QtObject {
     readonly property string windows: "windows"
     readonly property string linux: "linux"
     readonly property string mac: "osx"
+    readonly property string android: "android"
+    readonly property string ios: "ios"
```

Comment:
imo there is no need to exposed `android` and `ios` properties

Reply:
Uhm... might come handy at some point, like for the OSK handling